File: cs/rick.cs
using System;

namespace Rick
{
    public class Rick
    {
        public Rick Never()
        {
            Console.Write("Never ");
            return this;
        }
        public Rick Gonna()
        {
            Console.Write("gonna ");
            return this;
        }
        public Rick Give()
        {
            Console.Write("let ");
            return this;
        }
        public Rick You()
        {
            Console.Write("you ");
            return this;
        }
        public void Up() => Console.WriteLine("down!");
    }
    class Program
    {
        static void Main(string[] args)
        {
            Rick rick = new Rick();
            rick.Never().Gonna().Give().You().Up();
        }
    }
}

File: dart/rick.dart
import "dart:io";

class Rick {
    Rick never() {
        stdout.write("Never ");
        return this;
    }

    Rick gonna() {
        stdout.write("gonna ");
        return this;
    }

    Rick give() {
        stdout.write("let ");
        return this;
    }

    Rick you() {
        stdout.write("you ");
        return this;
    }

    void up() => print("down!");
}

void main() {
    Rick rick = Rick();
    rick.never().gonna().give().you().up();
}
